packages/react-examples/src/react-image/Image/Image.stories.tsx

```diff
@@ -60,6 +37,65 @@ export const ImageVariationsBorder = () => (
         width={200}
       />
     </Stack>
+  </FluentProvider>
+);
+
+export const ImageVariationsBorder = () => (
+  <FluentProvider theme={teamsLightTheme}>
+    <Stack horizontal>
+      <Stack horizontal>
+        <Image
+          alt="Ade's avatar"
```

Comment:
As we touch this area, let's use real names and avatars, you can grab them from https://github.com/microsoft/fluentui/pull/16486.